tiledb/sm/filesystem/hdfs_filesystem.h

```diff
@@ -39,164 +39,169 @@
 #include <string>
 #include <vector>
 
-#include "hdfs.h"
 #include "tiledb/sm/buffer/buffer.h"
 #include "tiledb/sm/misc/status.h"
 #include "tiledb/sm/misc/uri.h"
 #include "tiledb/sm/storage_manager/config.h"
 
+#include "hadoop/hdfs.h"
+
 namespace tiledb {
 namespace sm {
 
 namespace hdfs {
```

Comment:
Do we need this namespace anymore if HDFS is now a class?

Reply:
No, but when we add connection caching (for multiple connections) there will be another class in this namespace so I thought it better to leave it in for now.

Reply:
edit there is currently more than one class related to hdfs (`LibHDFS`, and `HDFS`)